unit-tests/dds/d405.py

```diff
@@ -38,181 +38,181 @@ def colored_infrared_stream():
-        # dds.video_stream_profile( 15, dds.stream_format("BGRA"), 480, 270 ),
-        # dds.video_stream_profile( 15, dds.stream_format("RGBA"), 480, 270 ),
-        # dds.video_stream_profile( 15, dds.stream_format("RGB2"), 480, 270 ),
-        # dds.video_stream_profile( 15, dds.stream_format("rgb8"), 480, 270 ),
-        dds.video_stream_profile( 5,  dds.stream_format("UYVY"), 480, 270 ),
-        # dds.video_stream_profile( 5,  dds.stream_format("BGRA"), 480, 270 ),
-        # dds.video_stream_profile( 5,  dds.stream_format("RGBA"), 480, 270 ),
-        # dds.video_stream_profile( 5,  dds.stream_format("RGB2"), 480, 270 ),
-        # dds.video_stream_profile( 5,  dds.stream_format("rgb8"), 480, 270 ),
-        dds.video_stream_profile( 90, dds.stream_format("UYVY"), 424, 240 ),
-        # dds.video_stream_profile( 90, dds.stream_format("BGRA"), 424, 240 ),
-        # dds.video_stream_profile( 90, dds.stream_format("RGBA"), 424, 240 ),
-        # dds.video_stream_profile( 90, dds.stream_format("RGB2"), 424, 240 ),
-        # dds.video_stream_profile( 90, dds.stream_format("rgb8"), 424, 240 ),
-        dds.video_stream_profile( 60, dds.stream_format("UYVY"), 424, 240 ),
-        # dds.video_stream_profile( 60, dds.stream_format("BGRA"), 424, 240 ),
-        # dds.video_stream_profile( 60, dds.stream_format("RGBA"), 424, 240 ),
-        # dds.video_stream_profile( 60, dds.stream_format("RGB2"), 424, 240 ),
-        # dds.video_stream_profile( 60, dds.stream_format("rgb8"), 424, 240 ),
-        dds.video_stream_profile( 30, dds.stream_format("UYVY"), 424, 240 ),
-        # dds.video_stream_profile( 30, dds.stream_format("BGRA"), 424, 240 ),
-        # dds.video_stream_profile( 30, dds.stream_format("RGBA"), 424, 240 ),
-        # dds.video_stream_profile( 30, dds.stream_format("RGB2"), 424, 240 ),
-        # dds.video_stream_profile( 30, dds.stream_format("rgb8"), 424, 240 ),
-        dds.video_stream_profile( 15, dds.stream_format("UYVY"), 424, 240 ),
-        # dds.video_stream_profile( 15, dds.stream_format("BGRA"), 424, 240 ),
-        # dds.video_stream_profile( 15, dds.stream_format("RGBA"), 424, 240 ),
-        # dds.video_stream_profile( 15, dds.stream_format("RGB2"), 424, 240 ),
-        # dds.video_stream_profile( 15, dds.stream_format("rgb8"), 424, 240 ),
-        dds.video_stream_profile( 5,  dds.stream_format("UYVY"), 424, 240 ),
-        # dds.video_stream_profile( 5,  dds.stream_format("BGRA"), 424, 240 ),
-        # dds.video_stream_profile( 5,  dds.stream_format("RGBA"), 424, 240 ),
-        # dds.video_stream_profile( 5,  dds.stream_format("RGB2"), 424, 240 ),
-        # dds.video_stream_profile( 5,  dds.stream_format("rgb8"), 424, 240 ),
-        dds.video_stream_profile( 5,  dds.stream_format("UYVY"), 256, 144 )
+        dds.video_stream_profile( 30, dds.video_encoding.uyvy, 1280, 720 ),
+        # dds.video_stream_profile( 30, dds.video_encoding("BGRA"), 1280, 720 ),
```

Comment:
I think that you can delete the commented profiles. The lines won't compile now if uncommented.

Reply:
Ahh... I see now that it will compile OK.
Up to you if to delete or not

Reply:
Removed